plugins/pulp_docker/plugins/token_util.py

```diff
@@ -82,12 +82,12 @@ def parse_401_response_headers(response_headers):
     """
     auth_header = response_headers.get('www-authenticate')
     if auth_header is None:
-        raise IOError("401 responses are expected to conatin authentication information")
+        raise IOError("401 responses are expected to contain authentication information")
     auth_header = auth_header[len("Bearer "):]
 
     # The remaining string consists of comma seperated key=value pairs
     auth_dict = {}
-    for key, value in (item.split('=') for item in auth_header.split(',')):
+    for key, value in (item.split('=') for item in auth_header.split('",')):
```

Comment:
i think we should specifically follow this https://tools.ietf.org/html/rfc6750#section-3 mentioned here https://docs.docker.com/registry/spec/auth/token/
going deeper https://tools.ietf.org/html/rfc2617 mentions that auth-params should be a quoted string.